src/gromo/growing_module.py

```diff
@@ -1183,32 +1183,58 @@ def reset_computation(self) -> None:
     def delete_update(
         self,
         include_previous: bool = True,
+        include_output: bool = False,
     ) -> None:
         """
         Delete the update of the optimal added parameters.
 
+        By default, we do not delete the extended_input_layer of this layer because it
+        could be used by the next layer.
```

Comment:
```suggestion
        By default, we do not delete the extended_output_layer of this layer because it
        could be required by the next layer.
```